### ISSUE TITLE
make own camera preview movable

### ISSUE THREAD

**Issue by 532910** [vector-im/riotX-android#1777]
Own camera preview should be movable same as in old riot-android

**Comment by Freepony** [comment]
Yes, I seconds that. I need that too because preview image often blocks the view and I have to ask people to move their cameras instead of move the preview myself.
